### ISSUE TITLE
How can i get the best model?

### ISSUE THREAD

**Issue by sticky-ai** [liznerski/fcdd#36]
Hi, thank you for your great work.

In the model training process, i wanna select the model with the highest performance. 
However, currently, the results learned in the last epoch are stored in the snapshot.

How can I get a best model?

**Comment by liznerski** [comment]
Typically, we don't have training labels for AD and thus also no validation set with which we could estimate the performance of the model during training. One could try to use synthetic anomalies or OE to get an estimate, but I'm not sure if that would generalize well. If you have some real anomalies available, you can use them for validation, but would have to remove them from the test set. 

Anyway, there is no such routine implemented, so you would have to add it to the Trainer ([somewhere here](https://github.com/liznerski/fcdd/blob/master/python/fcdd/training/bases.py#L204)).

**Comment by sticky-ai** [comment]
@liznerski Thank you for the quickly reply 👍
